sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/PreDefinedTagContainerTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.s
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Core.Tests
+{
+    public class PreDefinedTagContainerTests : ResourceManagerTestBase
+    {
+        public PreDefinedTagContainerTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        [OneTimeTearDown]
+        protected async Task GlobalTagCleanupAsync()
+        {
+            var container = Client.DefaultSubscription.GetPredefinedTag();
+            var operation = Client.GetPreDefinedTagsOperations();
+            var listResult = (await container.ListAsync().ToEnumerableAsync()).Where(x => x.Details.TagName.StartsWith("tagName"));
+            foreach (var item in listResult)
+            {
+                await item.DeleteAsync(item.Details.TagName).ConfigureAwait(false);
+            };
+        }
+
+        [Test, Order(1)]
+        [RecordedTest]
+        public async Task Create()
+        {
+            var tagName = Recording.GenerateAssetName("tagName");
+            var container = Client.DefaultSubscription.GetPredefinedTag();
+            var result = await container.CreateOrUpdateAsync(tagName);
+            Assert.IsTrue(result.Value.Details.TagName.Equals(tagName));
```

Comment:
I would expect this to be result.Value.Data...

Reply:
Done